tests/keras2onnx_applications/nightly_build/test_transformers.py

```diff
@@ -21,9 +21,20 @@
 if os.environ.get('ENABLE_FULL_TRANSFORMER_TEST', '0') != '0':
     enable_transformer_test = True
 
+def check_bloburl_access():
```

Comment:
I think we can add a common.py to save such common methods and avoid having duplicated ones in different tests files.

Reply:
Good point.